charts/gardener-extension-networking-calico/templates/clusterfilters-logging.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterFilter
+metadata:
+  labels:
+    fluentbit.gardener/type: "seed"
+  name: calico-node-calico-node
```

Comment:
As these resources are global, we need to achieve a unique name for each pod-container and that in naw achieved by unique pod-name-container-name combinations used in the `match` field. I just reuse it and combining them with `-`. So this means `podName-containerName`

Reply:
Ok, I understand the reasoning for the name, but I would propose to diverge from the default in this case and simply call it `calico-node` instead.
My reasoning is the following: You cannot have multiple calico installations in parallel in a cluster as the daemonsets would conflict due to statically configured ports in the host network.
The short name would be concise, obvious and understandable.
WDYT?

Reply:
Yes, I agree that in this way the name will look better. 
The only reason not to do that is that then this filter will defer from all other and it will look a bit strange when you run `kubectl get clusterfilters` and see ~40-50 filters with the same naming convention and just one which will be different

Reply:
Is it really like that? Do all the pods we monitor have a name using two components and a `-` character AND a container name with again two components and a `-` character in between?
I expect to have some differences in the names anyway if you use the approach to combine pod and container name. There should at least be some names with only three components.
Feel free to prove me wrong by sharing a list of examples.

Reply:
Also addressed